obs-browser-plugin.cpp

```diff
@@ -92,7 +92,10 @@ static void browser_source_get_defaults(obs_data_t *settings)
 	obs_data_set_default_int(settings, "fps", 30);
 	obs_data_set_default_bool(settings, "shutdown", false);
 	obs_data_set_default_bool(settings, "restart_when_active", false);
-	obs_data_set_default_string(settings, "css", default_css);
+    obs_data_set_default_string(settings, "css", default_css);
```

Comment:
Style:  It looks like the indentation here was changed from a tab to spaces.  Please revert this change, and use a tab to indent in the following lines.